CHANGELOG.md

```diff
@@ -1,1129 +1,1202 @@
-The new retreat system has been improved to take account of the profiles aggression. The higher the aggression the more enemy units needed to make the computer retreat. 
-Fixed an issue related to ghoul harvesting when wood rescource was low the attacking ghouls would be set to only 4 no matter what. This meant the ai's strength would also be incorrectly calculated as low. Now it is changed to be max 4 harvesting. The ghouls will still all harvest when no attack is running. 
-Fixed two amai script crashes when buying neutral units or items.
-Missed a defend town code that didn/t have the knew let it die detection features in
-Militia Expansion systems have been further improved and optimized. The militia sent will stay up there a build the hall. This dosn't include fast building it. That has yet to be optimized. 
-Town hall human fast building has been modulised into a job in preparation of making it work for any building you wish. (This is to allow humans to be extremly effective at quickly building up base hopefully in the future)
-Fixed an issue where illusions where being healed and treated as real units. This should prevent AMAI trying to save illusion units. 
-Militia expansion chance increased to 60% from 40% due to much smarter behaviour. 
-Fixed an issue with the functionality for using the moonstone.
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+
+## [Unreleased]
+
+## [2.6.1-cn] - 2022-08-4
+### Added
+- New chinese language translation, include Blizzard.eai [function Tribute]
+   - translation add more sentence for all language(translate by google)
+- Console add Page turning，adapted more than 12  AIPlayer and give more Strategy choose [Blizzard.eai]
+- if no HumanPlayer on game, the first Observer must choose language 
+- Add Undead UPGRADE UPG_SKEL_MASTERY
+- Undead strategy add GargGhouls
+- Input "-zoomxxxx" to adjust the sight distance, and XXXX is a number, such as "-zoom1000"，max zoom is 5000 (learn w3chambers)
+   - the Observer zoom increase 200
+
+### Changed
+- Computer skill level is now shown by default on AMAI players. (from [3.0.0]) 
+- Tweaked english gramatical errors for various strategy messages. (from [3.0.0]) 
+- Tweaks to Brewmaster and Alchemist skill picks. (Pixyy) (from [3.1.0]) 
+- Balance pass to all racial hero selection, skill selection and strategy selection. Mostly minor but most noticeable changes include (consult Pixyy)  (from [3.1.0]) 
+   - Night Elf increased use of warden and keeper.
+   - Reduced use of mirror image skill.
+- Strategy tweaks and optimizations  (consult Pixyy)  (from [3.1.0])
+   - Removed non-useful units from harass attacks and fixed undead harassment group numbering.
+   - Improvements to human, orc, undead and night elf dynamic strategy adjustment.
+- Strategy additional Improvements

```

Comment:
Note these strategy changes are missing at the moment. I'll merge what you got anyway as I can tidy up changelog to match later.